official-services.md

```diff
@@ -2,7 +2,6 @@
 
 The following services are officially operated by the FEniCS Project.
 
-- [Development mailing list](https://groups.google.com/forum/#!forum/fenics-dev)
```

Comment:
Has moved to slack.

Reply:
However, I just realized that that email address is the one used in every `pyproject.toml` file. If it has been discontinued, we need to use another email address there (the steering council one?).